gunicorn/config.py

```diff
@@ -346,6 +346,15 @@ def validate_pos_int(val):
     return val
 
 
+def validate_float(val):
+    if not val:
+        return 0.0
```

Comment:
no the  default should be the current one , so it doesn't break the current expectations ie 0.1 * random, which is still random where 0.05 isn't :)